themes/hugo-theme-docdock/layouts/partials/flex/head.html

```diff
@@ -1,7 +1,7 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
 {{ .Hugo.Generator }}
-<title>{{ .Title }} :: {{ .Site.Title }}</title>
+<title>{{ .Title }} - {{ .Site.Title }}</title>
```

Comment:
Do not modify source themes. It's like in wordpress. You never modify the theme itself. You create child themes. The `layouts/` folder is where the "child" themes go.